packages/material/scss/menu/_variables.scss

```diff
@@ -1,51 +1,149 @@
 
-$kendo-menu-item-expanded-bg: if($kendo-enable-color-system, k-color( base-active ), rgba( $kendo-menu-text, $kendo-flat-button-active-opacity * k-color-alpha( $kendo-menu-text ) )) !default;
-$kendo-menu-item-expanded-text: null !default;
-$kendo-menu-item-expanded-border: null !default;
-$kendo-menu-item-expanded-gradient: null !default;
+/// The background of active menu item.
+/// @group menu
+$kendo-menu-item-active-bg: if($kendo-enable-color-system, k-color( base-active ), rgba( $kendo-menu-text, $kendo-flat-button-active-opacity * k-color-alpha( $kendo-menu-text ) )) !default;
+/// The text color of active menu item.
+/// @group menu
+$kendo-menu-item-active-text: null !default;
+/// The border color of active menu item.
+/// @group menu
+$kendo-menu-item-active-border: null !default;
+/// The background gradient of active menu item.
+/// @group menu
+$kendo-menu-item-active-gradient: null !default;
 
+/// The base shadow of focused menu item.
+/// @group menu
 $kendo-menu-item-focus-shadow: null !default;
 
-$kendo-menu-separator-spacing: k-spacing(1) !default;
+/// The background of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-bg: null !default;
+/// The text color of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-text: null !default;
+/// The border color of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-border: null !default;
+/// The background gradient of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-gradient: null !default;
 
+/// The background of the menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-bg: $kendo-component-bg !default;
+/// The text color of the menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-text: $kendo-subtle-text !default;
+/// The border color of the menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-border: transparent !default;
+/// The background gradient of the menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-gradient: null !default;
 
+/// The background of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-bg: null !default;
+/// The text color of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-text: $kendo-menu-text !default;
+/// The border color of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-border: null !default;
+/// The background gradient of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-gradient: null !default;
 
 
+
+// Menu Separator
+
+/// The inline margin of the horizontal menu separator.
+/// @group menu
+$kendo-menu-separator-margin-inline: k-spacing(1);
+/// The block margin of the horizontal menu separator.
+/// @group menu
+$kendo-menu-separator-margin-block: k-spacing(1);
+/// The border color of  menu separator.
+/// @group menu
+$kendo-menu-separator-border: var(--kendo-color-border, #{$kendo-component-border}) !default;
```

Comment:
I think this should be changed to `if($kendo-enable-color-system, k-color(border), $kendo-component-border)`